pkg/dwarf/op/regs.go

```diff
@@ -52,10 +80,22 @@ func (regs *DwarfRegisters) Bytes(idx uint64) []byte {
 
 // Reg returns register idx or nil if the register is not defined.
 func (regs *DwarfRegisters) Reg(idx uint64) *DwarfRegister {
-	if idx >= uint64(len(regs.Regs)) {
-		return nil
+	if idx >= uint64(len(regs.regs)) {
+		if regs.loadMoreCallback != nil {
```

Comment:
nit: I think this would read better as:

```
if regs.loadMoreCallback == nil {
        return nil
}
regs.loadMoreCallback()
regs.loadMoreCallback = nil
if idx >= uint64(len(regs.regs)) {
	return nil
}
```